.github/workflows/automatic-updates.yml

```diff
@@ -0,0 +1,37 @@
+name: Automatically update Docker image versions
+
+on:
+  schedule:
+    - cron: "15 * * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'iasuhiu3hdiuwdhiuwhqfs'
```

Comment:
I assume this'll go away?

Reply:
Yes. This is there just to prevent the cronjob from running while the PR isn't ready.